cluster/gce/config-default.sh

```diff
@@ -546,3 +546,10 @@ PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
 # true if this is a tenant partition cluster
 KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
+
+if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == true ]]; then
+  ENABLE_APISERVER_INSECURE_PORT="${ENABLE_APISERVER_INSECURE_PORT:-false}"
+fi
+if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == true ]]; then
```

Comment:
same as above

Reply:
Please also think about use "KUBE_ENABLE_APISERVER_INSECURE_PORT:-false" when you set default value. KUBE_ENABLE_APISERVER_INSECURE_PORT is real parameter that we used in command line.

Reply:
kubemark use the KUBE_xxx to differentiate the variable set at the upper level of scripts and reserved xxx for internal usage to avoid naming conflicts. sometime this is overdone. i will see if we can consolidate the name